src/backend_common.cc

```diff
@@ -1356,4 +1358,17 @@ BufferAsTypedString(
   return nullptr;  // success
 }
 
+std::string
+GetRequestId(TRITONBACKEND_Request* request)
+{
+  const char* request_id = nullptr;
+  LOG_IF_ERROR(
+      TRITONBACKEND_RequestId(request, &request_id),
+      "unable to retrieve request ID string");
+  if ((request_id == nullptr) || (request_id[0] == '\0')) {
+    request_id = "<id_unknown>";
+  }
+  return std::string("[request id: ") + request_id + "]";
```

Comment:
`request->LogRequest()` in your core PR has a space at the end, but this one doesn't. Probably want to end with a space here in `GetRequestId` for consistency (and readability). 

Otherwise everything's going to look like:
`[request id: 123]other text goes here` with no space between ID and text.

Reply:
@GuanLuo do you think it makes more sense to put GetRequestId somewhere in common repo instead of backend so it can be used in server repo too?

It would be overloaded for both TRITONSERVER_Request* and TRITONBACKEND_Request* I think.

Reply:
Thanks for commenting. I'm happy to do either. Let me know (@rmccorm4 @GuanLuo) and I can make the changes, if needed.

Reply:
Sounds good. @dyastremsky just need the space from this comment I believe, otherwise LGTM:

> request->LogRequest() in your core PR has a space at the end, but this one doesn't. Probably want to end with a space here in GetRequestId for consistency (and readability).
>
> Otherwise I think everything's going to look like:
`[request id: 123]other text goes here` with no space between ID and text if I'm reading it correctly.